glotaran/builtin/megacomplexes/spectral/shape.py

```diff
@@ -12,49 +12,13 @@
         "amplitude": Parameter,
         "location": Parameter,
         "width": Parameter,
-        "skewness": {"type": Parameter, "allow_none": True},
     },
     has_type=True,
 )
-class SpectralShapeSkewedGaussian:
-    """A (skewed) Gaussian spectral shape"""
+class SpectralShapeGaussian:
+    """A Gaussian spectral shape"""
 
     def calculate(self, axis: np.ndarray) -> np.ndarray:
-        r"""Calculate a (skewed) Gaussian shape for a given ``axis``.
-
-        If a non-zero ``skewness`` parameter was added
-        :func:`calculate_skewed_gaussian` will be used.
-        Otherwise it will use :func:`calculate_gaussian`.
-
-        Parameters
-        ----------
-        axis: np.ndarray
-            The axis to calculate the shape for.
-
-        Returns
-        -------
-        shape: numpy.ndarray
-            A Gaussian shape.
-
-        See Also
-        --------
-        calculate_gaussian
-        calculate_skewed_gaussian
-
-        Note
-        ----
-        Internally ``axis`` is converted from :math:`\mbox{nm}` to
-        :math:`1/\mbox{cm}`, thus ``location`` and ``width`` also need to
-        be provided in :math:`1/\mbox{cm}` (``1e7/value_in_nm``).
-
-        """
-        return (
-            self.calculate_skewed_gaussian(axis)
-            if self.skewness is not None and not np.allclose(self.skewness, 0)
-            else self.calculate_gaussian(axis)
-        )
```

Comment:
This part is needed in the calculation of `SpectralShapeSkewedGaussian`, to prevent `ZeroDevisionError` in cases where `self.skewness` is close to zero.
If `SpectralShapeSkewedGaussian` would inherit from `SpectralShapeGaussian` you could use `super().calculate(axis)`.

Reply:
We should add a test with a small skewness where the initial parameter has the wrong sign and thus changes it during fitting.
Or at least a unit test for `SpectralShapeSkewedGaussian` with `self.skewness = 0` where the results are the same as for `SpectralShapeGaussian`.

Reply:
As mentioned numerous times: This is not a bug, it is a feature. In that case the user must switch to a gaussian model in that cases. The skew is a more advanced feature and people need to be aware of it. I also had this in discussion with @ism200 who agrees that we should not protect against this. But I will put in an Exception for this case.

Reply:
I added the exception 98b6b66

Reply:
I reverted to old behavior.